src/core/CircularProgress/index.js

```diff
@@ -4,28 +4,34 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import s from './styles'
 
+function Icon({ dashed }) {
```

Comment:
I think this name is too ambiguous. We already have an Icon. I know this one is only local to the file, but still. Perhaps just `Circle` or `SvgCircle` would be better.

Reply:
updated with _Circle_